lib/classes/Service.js

```diff
@@ -41,19 +41,33 @@ class Service {
       return BbPromise.resolve();
     }
 
+    let serverlessYamlPath = path.join(servicePath, 'serverless.yml');
```

Comment:
But the priority is now set to `.yml` extension now right? If it's not gonna take too long, you should probably change all those references, otherwise it's not super important. You've done an awesome job changing it everywhere else! 💥 👍